_includes/docs/user-guide/alarms.md

```diff
@@ -54,74 +58,196 @@ Assuming the following sequence of events:
-We can also optionally specify the relations that should be present between the parent entities and the originator for the alarm to propagate. 
+To simplify the database queries and improve load time, ThingsBoard supports the propagation of the alarm.
+We can specify whether an alarm should be visible for parent entities or not.
+We can also optionally specify the relations that should be present between the parent entities and the originator for the alarm to propagate.
+
+Alarm propagation settings are available in the *advanced settings* of the alarm rule of the device profile.
+
+{% include images-gallery.html imageCollection="propagation-settings" %}
+
+Let's look at an example of how it works. But first, we recommend reading the "[Getting Started](/docs/{{docsPrefix}}getting-started-guides/helloworld/)" guide. This will improve your understanding of ThingsBoard devices and dashboards and the concepts presented here.
+
+Let's say you have multiple offices with numerous devices in them. However, you only want to see alarms from devices that are associated with a specific office.
+Let's create a dashboard that will display alarms only from devices that are related to Office A.
+
+To find devices related to Office A, go to the "Assets" page, click on the needed asset and navigate to the "Relations" tab in the asset details window.
+The following devices relations to the Office A: Thermometer A1, Thermometer B1, Thermometer B2, and Thermometer C3.
+
+{% include images-gallery.html imageCollection="relations-to-asset" %}
+
+In order for the alarm created on the device to propagate further, in the alarm rule settings of the device profile, check the box "**Propagate alarm to related entities**". Additionally, you can specify relation types for alarm propagation.
+
+{% include images-gallery.html imageCollection="propagate-alarm-to-related-entities" %}
+
+Now, create a dashboard. Before adding a widget, create an alias that will filter the selected asset - "Office A".
+
+{% include images-gallery.html imageCollection="propagate-alias" %}
+
+Add the Alarms table widget. Specify the previously created alias as the alarm source.
+Be sure to activate the "**Search propagated alarms**" option.
+
+{% include images-gallery.html imageCollection="add-alarms-widget" %}
+
+Now, send telemetry to one of the devices that exceeds the threshold value specified in the alarm rule to trigger an alarm.
+
+{% include images-gallery.html imageCollection="propagate-send-telemetry" %}
+
+An alarm has been created on the device, and thanks to our settings, the alarm has propagated to the related asset. You can also see the created alarm on the "Alarms" tab   the asset details window.
+
+{% include images-gallery.html imageCollection="propagate-alarm-created" %}
 
-Now, when you know the theory, let's proceed to practical tutorials.       
+Now that you know the theory, let's proceed to practical tutorials.
 
-## Alarm FAQ and How-Tos
+## Create alarm rules
```

Comment:
Rename to "How to create alarms?" and explain that there are two options how one can create alarm rules. with links.